packages/core/config.d.ts

```diff
@@ -30,6 +30,15 @@ export interface Config {
      * @visibility frontend
      */
     title?: string;
+
+    /**
+     * The support url for the app.
```

Comment:
in the event this hierarchy is extended further, maybe this should be more generic, something like "Information about support of this Backstage instance and how to contact the integrator team." or something? that way later we could add back slack/discord/ms teams/persistent chat channel, and other mechanisms?

Reply:
The config has been re-worked and did my best to add meaningful descriptions and examples.